origins_workflow/origins_workflow.module

```diff
@@ -336,7 +336,7 @@ function origins_workflow_preprocess_table(&$variables) {
           // Check the existing text in an attempt to detect any core fix that
           // might come along.
           $title_text = $title_markup->getUntranslatedString();
-          if (($title_text == 'Set as current revision') || ($title_text == 'Revert')) {
+          if ($title_text == 'Set as current revision') {
```

Comment:
Decided that there is no need to replace the core Drupal 'Revert' functionality as it is working correctly, it is only the 'Set as current revision' behaviour that we actually need to override.